Bourreau/lib/bourreau_system_checks.rb

```diff
@@ -484,6 +485,44 @@ def self.a100_ensure_dp_cache_symlink_exists #:nodoc:
 
 
 
+  # touch to avoid deletion by cluster bimonthly sctratch cleanup : 1) DataProvider cache dir, 2) DP_Cache_Key.md5 3) DP_Cache_Rev.id
+  # 4) gridshare dir, and 5) DP_Cache symbolic link located in it
+  # gridshare and cache_dir are typically updated often, but we touch them just in the case
+  def self.a105_ensure_dp_cache_and_symlink_will_exists #:nodoc:
```

Comment:
1. I think this new checker will prevent a new CBRAIN installation from booting, given that it probably doesn't have a DP cache directory already configured. We allow a CBRAIN portal to start even without one, because we expect the admin to configure it with the interface.
2. most of the code in here should be moved to the DataProvider class, alongside other methods that generate/install these files; the checker code should be made way simpler (just report if yes/no the update was performed).